### PR TITLE
Add edit function

### PR DESCRIPTION
**Pull request by cmaiw**
Still work in Progress

### CODE REVIEW

File: client/src/pages/EditEntry.js [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+`;
+
+function EditEntry() {
+  const [edit, setEdit] = React.useState([]);
+  const { editId } = useParams();
+  const history = useHistory();
+
+  async function saveEditedEntriesToDB(value) {
+    setEdit();
+    await fetch(`/api/entries${editId}`, {
```
Comment (lmachens) [inline client/src/pages/EditEntry.js]:
```suggestion
    await fetch(`/api/entries/${editId}`, {
```

I think  the URL is invalid without the slash

File: client/src/pages/EditEntry.js [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+  flex-direction: row;
+  width: 100%;
+  margin: 4px;
+  justify-content: flex-start;
+  align-items: space-around;
+  margin-top: 8px;
+`;
+
+function EditEntry() {
+  const [edit, setEdit] = React.useState([]);
```
Comment (lmachens) [inline client/src/pages/EditEntry.js]: You are mixing the type of edit in this file.
The initial value is an empty array. 
In the onChange, you change it to an object. 
In the fetch PATCH, you use it to set the name.

Comment (cmaiw) [reply client/src/pages/EditEntry.js]: Changed this after refactoring, thank you.

File: client/src/pages/EditEntry.js (hunk 1 above)
Comment (cmaiw) [reply client/src/pages/EditEntry.js]: Thank you.

**Pull request merged**